driver_unix.go

```diff
@@ -4,11 +4,27 @@ package continuity
 
 import (
 	"fmt"
+	"os"
 	"sort"
+	"syscall"
 
 	"github.com/stevvooe/continuity/sysx"
 )
 
+const (
+	atNoFollow = 0x100
+)
+
+func (d *driver) Unlink(path string) error {
+	return syscall.Unlink(path)
+}
+
+// Lchmod changes the mode of an file not following symlinks. The name
+// must be an absolute path to a file.
+func (d *driver) Lchmod(name string, mode os.FileMode) error {
+	return syscall.Fchmodat(0, name, uint32(mode), atNoFollow)
```

Comment:
From linux kernel description
>  If pathname is absolute, then dirfd is ignored

From BSD description (could not find directly in darwin, maybe darwin has something akin to lchmod)
> The fchmodat() is equivalent to either chmod() or lchmod()	depending on the flag except in the case where path specifies a relative path.

Fchmodat should operate the same as `chmod` when the given fd is 0. There is a comment about how this function must take in an absolute path. I could explicitly check and error here if it is not. On linux there is `lchmod` so this is the correct system call to use.